rfcs/0064-documentation-format.md

```diff
@@ -0,0 +1,142 @@
+---
+feature: documentation-format
+start-date: 2019-12-31
+author: Silvan Mosberger (infinisil)
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+The Nix community wants to move away from using Docbook as our documentation format and replace it with something else. However it is unclear what it should be replaced with. This RFC gives a concrete process for determining the new documentation format. It does NOT say what format should be used.
+
+# Motivation
+[motivation]: #motivation
+
+There's been enough bike-shedding over the documentation format to use. We should finally settle this debate by deciding and committing on a single format such that we can move forward and improve our documentation situation.
+
+# Detailed design
+[design]: #detailed-design
+
+The process for determining the doc format is as follows:
+- A set of requirements for the doc format is decided through the RFC discussion
+- Doc format candidates are collected and evaluated to see if they fulfil the requirements.
+- A short objective overview of each valid candidate format is written, along with their advantages/disadvantages
+- The RFC is accepted
+- A [Discourse](https://discourse.nixos.org/) post is created with these overviews, along with a poll such that people can vote on the formats they prefer. This poll will be open to the whole community and should be advertised as such
+- Whatever format wins in the poll is chosen as the new default documentation format. If later it is discovered that the winner is infeasible for any reason, e.g. if it doesn't meet the requirements after all, the format on second place is chosen instead, and so on.
+
+## Poll
```

Comment:
I think poll is too vague, people should discuss on this RFC exactly what they'd like and then the shepherd team makes a decision.

Reply:
I think it's important to involve as many people from the community as possible, because they will be the ones writing and reading the docs. Unfortunately RFC's are known to be bad places for involving many people, since they tend to get long quickly and less-active people get overshadowed by very vocal people.

As you suggested to me though, having the poll happen before the RFC is accepted and using it as an input for the shepherd team might be a good idea.

Reply:
I changed the RFC to reflect this

Reply:
> as many people from the community as possible
sorry to be picky and little offtopic, but how you are going to handle potentially hundreds of people?